BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+ExchangeRateService service = new ExchangeRateService();
+
+// REST API to get exchange rates based on input currencies
+app.MapPost("/", service.GetExchangeRates);
```

Comment:
- I would consider creating a Controller to deal with the request body (the currencies informed by the client), and to redirect it to the **IExchangeRateService**, which will be responsible for calling the external service to obtain the exchange rates.
- In order to adhere to the RESTful API principles, we should use the **GET** HTTP method to retry information, since we should only use POST when the intent is to create a new object.
- Add Swagger to the controller to generate documentation: https://learn.microsoft.com/en-us/aspnet/core/tutorials/getting-started-with-swashbuckle?view=aspnetcore-8.0&tabs=visual-studio